lib/kafo/parser_cache_reader.rb

```diff
@@ -38,20 +40,30 @@ def self.logger
       KafoConfigure.logger
     end
 
-    def initialize(cache)
+    def initialize(cache, options = {})
       @cache = cache
+      @force = options[:force]
     end
 
     def logger
       KafoConfigure.logger
     end
 
     def get(key, manifest_path)
+      if @force == false
+        logger.debug "Skipping parser cache for #{manifest_path}, forced off"
+        return nil
+      end
+
       return nil unless @cache[:files].has_key?(key)
 
       if @cache[:files][key][:mtime] && File.mtime(manifest_path).to_i > @cache[:files][key][:mtime]
-        logger.debug "Parser cache for #{manifest_path} is outdated, ignoring cache entry"
-        return nil
+        if @force
+          logger.debug "Parser cache for #{manifest_path} is outdated, forced to use it anyway"
```

Comment:
if this should be used with caution, would it make more sense to use warning instead of debug log level here?